dotnet/src/IntegrationTests/testsettings.json

```diff
@@ -51,8 +51,8 @@
     "EmbeddingModelId": "embedding-001",
     "ApiKey": "",
     "Gemini": {
-      "ModelId": "gemini-1.0-pro",
-      "VisionModelId": "gemini-1.0-pro-vision"
+      "ModelId": "gemini-1.5-flash",
+      "VisionModelId": "gemini-1.5-flash"
```

Comment:
Does it mean that the new change is not compatible with `gemini-1.0-pro` model? If it's not compatible, how would it work for customers who use that model? If it is compatible, why do we need to change model in integration tests?

Reply:
It doesn't work with gemini-1.0-pro but end user will get a 400 with message "Developer instruction is not enabled for models/gemini-1.0-pro". For existing customers, it shouldn't matter because they couldn't use system messages before. And beside that google nuget is still alpha.